code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1144,6 +1144,13 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 			H.update_inv_w_uniform()
 			H.update_inv_wear_suit()
 
+	if(H.noisy && H.nutrition <= NUTRITION_LEVEL_HUNGRY)
+		if(prob(10))
+			playsound(get_turf(H),"hunger_sounds",15,0,-5,1,ignore_walls = FALSE,channel=CHANNEL_PRED)
+	else if(H.noisy && H.nutrition <= NUTRITION_LEVEL_STARVING)
```

Comment:
shouldnt these be flipped? starving is a lower value than hungry

Reply:
Suppose they could be flipped, yee.